hybrid/flow.py

```diff
@@ -154,12 +156,11 @@ def halt(self):
 
 
 class RacingBranches(Runnable, traits.SIMO):
-    """Runs (races) multiple :class:`Branch`es in parallel, but stops all others
-    after the first one finishes. Returns the results of all branches, in order
-    specified.
+    """Runs (races) multiple :class:`Branch` classes in parallel, but stops all others
```

Comment:
Instead of "classes" I would say "objects". Or maybe even "runnables".

Actually, we should be more general than that. We are racing complete *workflows* (runnables) here.

Reply:
"Runs (races) multiple workflows of :class:`~hybrid.core.Runnable` classes in parallel, stopping all once the first finishes. Returns the results of all, in the specified order."

Reply:
With *type* in place of *class* sounds more correct:
> Runs (races) multiple workflows of type :class:~hybrid.core.Runnable in parallel, stopping all once the first finishes. Returns the results of all, in the specified order.

Reply:
Changed. The PR now has a merge conflict, but that's just where you changed LoopN, so I'll leave it to you rather than rebase and update the PR. Thanks!